Python/compile.c

```diff
@@ -3310,18 +3308,16 @@ compiler_try_star_finally(struct compiler *c, stmt_ty s)
    []                           POP_BLOCK
    []                           JUMP_FORWARD    L0
 
-   [tb, val, exc]       L1:     <evaluate E1>                   )
-   [tb, val, exc, E1]           JUMP_IF_NOT_EXC_MATCH L2        ) only if E1
-   [tb, val, exc]               POP
-   [tb, val]                    <assign to V1>  (or POP if no V1)
-   [tb]                         POP
+   [exc]                        L1:     <evaluate E1>           )
```

Comment:
```suggestion
   [exc]                L1:     <evaluate E1>           )
```

Lines +3317 and +3320 show that a label should be in its own column:
```
   [exc]                L2:     <evaluate E2>
```
```
   [exc]                Ln+1:   RERAISE     # re-raise exception
```
I think that indentation of `Lx:` should be the same.